app/assets/stylesheets/application.scss

```diff
@@ -52,3 +52,25 @@ thead {
 .managing_editor_note {
   margin-left: 20px;
 }
+
+
+.review-anonymous-feedback {
+  .mark-all-spam {
+    text-align: center;
+  }
+
+  .table > tbody > tr > td.mark-as-spam-checkbox {
+    text-align: center;
+    vertical-align: middle;
+  }
+
+  .mark-as-spam-checkbox label {
+    display: block;
+    padding: 12px;
+  }
+
+  .problem-report.reviewed,
+  .problem-report.reviewed .table-header-secondary {
+    background-color: #f4fef4;
```

Comment:
I know we're only using this in a single place, but can we set it up as a sass variable (`$light-green`) similar to how govuk_admin_template sets it up? Perhaps in `application.scss` -- see https://github.com/alphagov/govuk_admin_template/blob/master/app/assets/stylesheets/govuk_admin_template/_theme.scss